nucypher/policy/policies.py

```diff
@@ -277,6 +277,22 @@ def publish(self, network_middleware: RestMiddleware) -> dict:
         """
         return self.publish_treasure_map(network_middleware=network_middleware)
 
+    def credential(self, with_treasure_map=True):
```

Comment:
I think `make_credential` or similar is more descriptive here, since we are not *using* the credential here.

Reply:
I like more concise option. Or what, in your opinion, does the `credential()` method name say intuitively?

Reply:
Credential is already a verb. When you _credential_ someone, you are giving them credential.

Reply:
Using the verb form of `credential` sounds to my ear like you are *presenting it to gain access*, not merely possessing it.  `Policy` can be used to produce a credential, but It sounds  to me like `Alice` / `Bob` do the actual credentialing.